plutus-core/plutus-ir/test/prettyprintingReadable/even3Eval

```diff
@@ -0,0 +1,29 @@
+(let (rec)
```

Comment:
this is the same test as even3 :) we just actually evaluate it in the other contexts. no point duplicating it here